workflows/cve-fixer/.claude/commands/cve.find.md

```
@@ -51,59 +51,40 @@ Report: artifacts/cve-fixer/find/cve-issues-20260226-145018.md
+       export JIRA_EMAIL="your-email@redhat.com"
+       ```
+   - **HTTP 403** → token valid but insufficient permissions — inform user
+   - **Other / timeout** → network issue — inform user and retry once
+
+   **Do NOT pre-check env vars with `[ -z "$JIRA_API_TOKEN" ]` and stop.** The variables may be available to the API call even if not visible to the shell check (e.g. Ambient secrets injection).
 
 3. **Query Jira for CVE Issues**
 
-   a. Set up variables:
+   a. Set up variables (AUTH already set from Step 2):
    ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)